frameworks/helloworld/tests/test_sanity.py

```diff
@@ -32,12 +32,14 @@ def configure_package(configure_security):
     try:
         sdk_install.uninstall(FOLDERED_SERVICE_NAME, package_name=PACKAGE_NAME)
 
+        service_config = {"service": {"name": FOLDERED_SERVICE_NAME, "user": "root"}}
```

Comment:
Create a ticket for changing the version 1.8 in the package.json. Assign to me.

We'll then naturally produce a HW package at the end of the sprint.